### ISSUE TITLE
Add new Hawaii exclude polygon

### ISSUE THREAD

**Issue by amedenblik** [USGS-WiM/StreamStats#1180]
Jack Mitchell sent a new set of exclusion polygons for Hawaii data on the Dev server. They need to be added to the exclude poly map services for Dev.

[new_HI_excl_pol.zip](https://github.com/USGS-WiM/StreamStats/files/8034647/new_HI_excl_pol.zip)

**Comment by amedenblik** [comment]
I added the new Hawaii exclude poly to stateServices_dev. When attempting to publish the service, I got this error: "Error 001270 consolidating the data failed". I went to Customize > ArcMap Options > Sharing and temporarily changed the "Staging Path" from `C:\Users\amedenblik\AppData\Local\Esri\Desktop10.5\Staging` to `D:\staging` (a temp folder I created). This resolved the error. The error may be due to the fact that the C drive is almost full. 

I will contact Ryan. After his confirmation, I will update stateServices_test.
